verification/curator-service/api/test/model/schedule.test.ts

```diff
@@ -25,6 +25,24 @@ describe('validate', () => {
         });
     });
 
+    it('a schedule without an AWS schedule expression is invalid', async () => {
+        const missingExpression = { ...fullModel };
+        delete missingExpression.awsScheduleExpression;
+
+        return new Schedule(missingExpression).validate((e) => {
+            expect(e.name).toBe(Error.ValidationError.name);
+        });
+    });
+
+    it('a schedule with a misformated AWS schedule expression is invalid', async () => {
+        const badSchedule = { ...fullModel };
```

Comment:
up to you, can inline as

`return new Schedule({ ...fullModel, awsScheduleExpression: 'rate(1 hour') }).validate ...`

but totally get it if you find that less readable

Reply:
Ah, that's pretty nice! I'll shelf this for after the schema changes settle. This existing format is probably used in most tests.